src/utils/generateUsageSnippets.js

```diff
@@ -0,0 +1,32 @@
+import { FEATURE_SETS_TAB, FEATURE_VECTORS_TAB } from '../constants'
+
+export const generateUsageSnippets = pageTab => {
+  if (pageTab === FEATURE_SETS_TAB) {
+    return {
+      title: 'Get offline features for training:',
+      code: `features = [
+              "<feature set>.*",
+             ]
+             vector = FeatureVector(features=features)
+             resp = get_offline_features(
+              vector, entity_rows=<entity>, entity_timestamp_column="<entity timestamp column"
+             )
+             print(resp.to_dataframe())
+             print(vector.get_stats_table())
+             resp.to_parquet("./out.parquet")`
+    }
+  }
+
+  if (pageTab === FEATURE_VECTORS_TAB) {
+    return {
+      title: 'Getting online features:',
+      code: `svc = get_online_feature_service(vector_uri)
+              resp = svc.get([{"key": "value"}, {"key": "value"}])
+              print(resp)
+              
+              
+              resp = svc.get([{"key": "value"}], as_list=True)
+              print(resp)`
```

Comment:
I suggest this change:
```suggestion
resp = svc.get([{"key": "value"}, {"key": "value"}])
print(resp)

resp = svc.get([{"key": "value"}], as_list=True)
print(resp)`
```

so it looks like this:
![image](https://user-images.githubusercontent.com/13918850/114916039-020c2780-9e2d-11eb-8887-12affdf673f5.png)

instead of this:
![image](https://user-images.githubusercontent.com/13918850/114916142-1d773280-9e2d-11eb-8b11-23df860d5bed.png)